Sources/MapboxMaps/Foundation/Camera/CameraAnimator.swift

```diff
@@ -1,20 +1,38 @@
 import UIKit
+import CoreLocation
 
-// MARK: CameraAnimator Class
-public class CameraAnimator: NSObject {
+@objc public protocol CameraAnimatorProtocol: AnyObject {
+
+    /// Stops the animation in its tracks and calls any provided completion
+    func stopAnimation()
 
-    // MARK: Stored Properties
+    /// The current state of the animation
+    var state: UIViewAnimatingState { get }
+
+    /// TODO: Move this to a non-public interface
+    func update()
+}
+
+// MARK: CameraAnimator Class
+public class CameraAnimator: NSObject, CameraAnimatorProtocol {
 
     /// Instance of the property animator that will run animations.
-    private var propertyAnimator: UIViewPropertyAnimator
+    internal var propertyAnimator: UIViewPropertyAnimator
 
     /// Delegate that conforms to `CameraAnimatorDelegate`.
-    private weak var delegate: CameraAnimatorDelegate?
+    internal weak var delegate: CameraAnimatorDelegate?
 
     /// The ID of the owner of this `CameraAnimator`.
-    internal var owner: AnimationOwner
+    public internal(set) var owner: AnimationOwner
+
+    /// The `CameraView` owned by this animator
+    internal var cameraView: CameraView
 
-    // MARK: Computed Properties
+    /// Represents the animation that this animator is attempting to execute
+    internal var animation: ((inout CameraTransition) -> Void)?
+
+    /// Defines the transition that will occur to the `CameraOptions` of the renderer due to this animator
+    public internal(set) var transition: CameraTransition?
 
     /// The state from of the animator.
     public var state: UIViewAnimatingState { return propertyAnimator.state }
```

Comment:
for 1-liners, I'm in favor of omitting `return`

Reply:
Sounds good.